### DISCUSSION TITLE
Missing forEachObject and getObjects methods in Group in v6?

### DISCUSSION THREAD

**Discussion by lukaszkowalik2** [fabricjs/fabric.js#9535] [General]
In version v5 of the fabric.js library, the Group object had convenient methods like `forEachObject` and `getObjects`. However, in the transition to v6, these methods seem to be missing. The absence of these methods has led to a workaround, but I am uncertain if it is the optimal solution.

**Current Workaround:**
```javascript
canvas.forEachObject((obj) => {
  const objects = obj.get("_objects");
  objects.forEach((ob) => {
    console.log(ob);
  });
});
```

**Concerns:**

The current workaround involves accessing private properties (_objects), which may not be a sustainable or recommended approach. Is this the preferred way to iterate through objects in a fabric.js Group in version v6, or is there a better alternative?

**Request for Guidance:**
I would appreciate guidance on whether the current workaround is a suitable solution or if there is a recommended method to achieve the same result in fabric.js v6.

**CodeSandbox Link:**
[CodeSandbox - fabric.js v6 Group Issue](https://codesandbox.io/p/devbox/youthful-ace-8tnxyp?file=%2Fsrc%2FApp.tsx%3A32%2C1-37%2C10)

**Comment by ShaMan123** [comment]
Where did you get that impression?
Group has those methods
https://codesandbox.io/p/devbox/practical-almeida-gkcx8j?file=%2Fsrc%2FApp.tsx%3A33%2C24
